docs/guides/quickstarts/react-native.mdx

```diff
@@ -0,0 +1,412 @@
+---
```

Comment:
the quickstart includes sign in with Apple/Google but there is no mention of it here and it won't just work out of the box. I think we should add a "Next Steps" section at the end pointing to the guides for this indicating that those won't work just yet and they need to be configured.